cmd/testworkflow-init/commands/run.go

```diff
@@ -0,0 +1,74 @@
+package commands
+
+import (
+	"slices"
+
+	"github.com/kubeshop/testkube/cmd/testworkflow-init/constants"
+	"github.com/kubeshop/testkube/cmd/testworkflow-init/data"
+	"github.com/kubeshop/testkube/cmd/testworkflow-init/orchestration"
+	"github.com/kubeshop/testkube/cmd/testworkflow-init/output"
+	"github.com/kubeshop/testkube/pkg/expressions"
+	"github.com/kubeshop/testkube/pkg/testworkflows/testworkflowprocessor/action/actiontypes/lite"
+)
+
+func Run(run lite.ActionExecute, container lite.LiteActionContainer) {
+	machine := data.GetInternalTestWorkflowMachine()
+	state := data.GetState()
+	step := state.GetStep(run.Ref)
+
+	// Abandon executing if the step was finished before
+	if step.IsFinished() {
+		return
+	}
+
+	// Obtain command to run
+	command := make([]string, 0)
+	if container.Config.Command != nil {
+		command = slices.Clone(*container.Config.Command)
+	}
+	if container.Config.Args != nil {
+		command = append(command, *container.Config.Args...)
+	}
+
+	// Resolve the command to run
+	for i := range command {
+		value, err := expressions.CompileAndResolveTemplate(command[i], machine, expressions.FinalizerFail)
+		if err != nil {
+			output.ExitErrorf(data.CodeInternal, "failed to compute argument '%d': %s", i, err.Error())
+		}
+		command[i], _ = value.Static().StringValue()
+	}
+
+	// Run the operation
+	execution := orchestration.Executions.Create(command[0], command[1:])
```

Comment:
command := make([]string, 0)
	if container.Config.Command != nil 

sounds  like command can be empty